更新翻译提交 (#15)
NucoTech/iris-doc: GettingStarted/InstallingIris.md

> # 下载Iris(Installation)

Reviewer: ```suggestion
# 下载(Installation)
```

Iris是一个跨平台框架。  

> 运行需要go语言的环境, 要求版本在1.14及以上。

Reviewer: ```suggestion
唯一需求是[Go语言](https://golang.org/dl/), 要求版本在1.14及以上。
```

```shell
go env -w GO111MODULE=on
```

## 安装(Install)

```shell
go get github.com/kataras/iris/v12@v12.1.8
```

或者可以编辑你项目的`go.mod`文件。

> ```go

Reviewer: ```suggestion
```go.mod
```

module your_project_name

go 1.14

require (
    github.com/kataras/iris/v12 v12.1.8
)
```

```shell
go build
```

> ## 常见问题(TroubleShooting)

Reviewer: ```suggestion
## 故障排除
```

> 如果你在安装`iris`的时候遇到了错误, 请确保你的[`GOPROXY`环境变量](https://github.com/golang/go/wiki/Modules#are-there-always-on-module-repositories-and-enterprise-proxies)设置为一个有效的值

Reviewer: ```suggestion
如果你在安装`iris`的时候遇到了错误, 请确保你设置了合法的[`GOPROXY`环境变量](https://github.com/golang/go/wiki/Modules#are-there-always-on-module-repositories-and-enterprise-proxies)
```
